linkerd/app/core/src/classify.rs

```diff
@@ -183,10 +183,9 @@ fn grpc_class(headers: &http::HeaderMap) -> Option<Class> {
         .and_then(|v| v.to_str().ok())
         .and_then(|s| s.parse::<u32>().ok())
         .map(|grpc_status| {
-            let ok = if grpc_status == 0 {
-                SuccessOrFailure::Success
-            } else {
-                SuccessOrFailure::Failure
+            let ok = match grpc_status {
+                0 | 3 | 5 => SuccessOrFailure::Success,
```

Comment:
It can be hard to remember what the numbers mean exactly. How about a comment here reminding us?

Reply:
Are we able to use the const's from the gRPC crate? https://github.com/tower-rs/tower-grpc/blob/master/tower-grpc/src/status.rs#L26

Reply:
Sure, I was going to ask what did you think about going this way:

```rust
let ok = match grpc::Code::from_i32(grpc_status as i32) {
    grpc::Code::Ok | grpc::Code::InvalidArgument | grpc::Code::NotFound => {
        SuccessOrFailure::Success
    }
    _ => SuccessOrFailure::Failure,
};
```

I saw the `0` before and I intended to follow the same approach. First contact with Rust, any feedback is more than welcome :)

Reply:
I think we may not have had those consts available to us when this code was initially written :)  This change seems good to me.

Reply:
Thanks 👍